scripts/performance/perf_plugins.py

```diff
@@ -292,11 +292,21 @@ def _set_nvl_domain_size(
         executor: "run.Executor",
         moe_flex_dispatcher_backend: str,
         gpu: str,
+        model_recipe_name: str,
+        model_family_name: str,
     ):
         if moe_flex_dispatcher_backend == "hybridep" and gpu in ["gb200", "gb300"]:
             executor.env_vars["NVLINK_DOMAIN_SIZE"] = "72"
             executor.env_vars["NUM_OF_HYBRID_EP_RANKS_PER_NVLINK_DOMAIN"] = "8"
             executor.env_vars["USE_MNNVL"] = "1"
+        if (
+            moe_flex_dispatcher_backend == "hybridep"
+            and (gpu == "gb300" and model_recipe_name == "qwen3_235b_a22b")
+            or (gpu == "gb300" and model_recipe_name == "deepseek_v3")
+        ):
+            executor.env_vars["NVLINK_DOMAIN_SIZE"] = "72"
+            executor.env_vars["NUM_OF_HYBRID_EP_RANKS_PER_NVLINK_DOMAIN"] = "64"
```

Comment:
Let's modify this to something like

```
executor.env_vars["NUM_OF_HYBRID_EP_RANKS_PER_NVLINK_DOMAIN"] = ep_size
```
Since EP is placed at the lowest network domain, we can simply add an assertion that ep_size > 72.